src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1549367836.inc.php

```diff
@@ -0,0 +1,319 @@
+<h1>Build #1549367836</h1>
+<h2>Date: 2019-02-05</h2>
+<div class="changelog">
+    - Add CMS main menu item table.
+</div>
+<?php
+
+// Add cms_menu_item table.
+
+$cmsMenuItemTableId = TCMSLogChange::createUnusedRecordId('cms_tbl_conf');
+
+$query = "CREATE TABLE `cms_menu_item` (
+                  `id` CHAR( 36 ) CHARACTER SET latin1 COLLATE latin1_general_ci NOT NULL ,
+                  `cmsident` INT UNSIGNED NOT NULL AUTO_INCREMENT COMMENT 'Key is used so that records can be easily identified in Chameleon',
+                  PRIMARY KEY ( `id` ),
+                  UNIQUE (`cmsident`)
+                ) ENGINE = InnoDB";
+TCMSLogChange::RunQuery(__LINE__, $query);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_tbl_conf', 'en')
+  ->setFields([
+      'name' => 'cms_menu_item',
+      'dbobject_type' => 'Customer',
+      'translation' => 'CMS main menu items',
+      'engine' => 'InnoDB',
+      'list_query' => '',
+      'cms_content_box_id' => '',
+      'only_one_record_tbl' => '0',
+      'is_multilanguage' => '0',
+      'is_workflow' => '0',
+      'locking_active' => '0',
+      'changelog_active' => '0',
+      'revision_management_active' => '0',
+      'name_column' => '',
+      'name_column_callback' => '',
+      'display_column' => '',
+      'display_column_callback' => '',
+      'list_group_field' => '',
+      'list_group_field_header' => '',
+      'list_group_field_column' => '',
+      'cms_tbl_list_class_id' => '',
+      'table_editor_class' => '',
+      'table_editor_class_subtype' => '',
+      'table_editor_class_type' => 'Core',
+      'icon_list' => '',
+      'show_previewbutton' => '0',
+      'cms_tpl_page_id' => '',
+      'rename_on_copy' => '0',
+      'cms_usergroup_id' => '6',
+      'notes' => '',
+      'frontend_auto_cache_clear_enabled' => '1',
+      'dbobject_extend_class' => 'TCMSRecord',
+      'dbobject_extend_subtype' => 'dbobjects',
+      'dbobject_extend_type' => 'Core',
+      'auto_limit_results' => '-1',
+      'icon_font_css_class' => '',
+      'id' => $cmsMenuItemTableId,
+  ])
+;
+TCMSLogChange::insert(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_tbl_conf', 'de')
+  ->setFields([
+      'translation' => 'CMS Hauptmenüpunkt',
```

Comment:
"Hauptmenüpunkt" is the translation of "main menu items"?

Reply:
Changed to use plural.